### ISSUE TITLE
name: 🐛 Bug Report
description: Create a report to help us reproduce and fix the bug

body:
  - type: markdown
    attributes:
      value: >
        #### Before submitting a bug, please make sure the issue hasn't been already addressed by searching through [the
        existing and past issues](https://github.com/meta-llama/llama-stack/issues).

  - type: textarea
    id: system-info
    attributes:
      label: System Info
      description: |
        Please share your system info with us. You can use the following command to capture your environment information
        python -m "torch.utils.collect_env"

      placeholder: |
        PyTorch version, CUDA version, GPU type, #num of GPUs...
    validations:
      required: true

### ISSUE THREAD

**Issue by Lokmanmama** [meta-llama/llama-stack#415]

**Comment by Lokmanmama** [comment]
> 

Goog
